sale_deposit_clear_option/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds an option to clear down payments on sale order documents.
```

Comment:
It should be great to describe a little bit more the process as it seems not obvious at first sight.